docs/platforms/python/tracing/instrumentation/custom-instrumentation/index.mdx

```diff
@@ -69,15 +69,32 @@ def eat_pizza(pizza):
     with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
         while pizza.slices > 0:
             eat_slice(pizza.slices.pop())
-
 ```
 
-<Alert title="Static & Class Methods" level="warning">
+<Alert title="Static & Class Methods" level="info">
 
 When tracing a static or class method, you **must** add the `@sentry_sdk.trace` decorator **after** the `@staticmethod` or `@classmethod` decorator (i.e., **closer** to the function definition). Otherwise, your function will break!
 
 </Alert>
 
+### Manually
+
+```python
+import sentry_sdk
+
+def eat_slice(slice):
+    ...
+
+def eat_pizza(pizza):
+    with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
+        while pizza.slices > 0:
+            span = sentry_sdk.start_span(description="Eat Slice")
+            eat_slice(pizza.slices.pop())
+            span.finish()
+```
+
+When you create your span manually, make sure to call `span.finish()` after the block of code you want to wrap in a span to finish the span. If you do not finish the span it will not be sent to Sentry.
+
 ## Nested Spans
 
 Spans can be nested to form a span tree. If you'd like to learn more, read our [distributed tracing](/product/sentry-basics/tracing/distributed-tracing/) documentation.
```

Comment:
I don't think the examples in this section are needed, we could probably delete them